client/src/reducers/controls.ts

```diff
@@ -37,13 +39,31 @@ const minimizeMaximizePopUps = (
   };
 };
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
+interface ControlsState {
+  loading: boolean;
+  error: Error | string | null;
+  resettingInterface: boolean;
+  graphInteractionMode: string;
+  opacityForDeselectedCells: number;
+  scatterplotXXaccessor: string | boolean;
+  scatterplotYYaccessor: string | boolean;
+  geneIsOpen: boolean;
+  scatterplotIsMinimized: boolean;
+  geneIsMinimized: boolean;
+  scatterplotLevel: string;
+  scatterplotIsOpen: boolean;
+  geneLevel: string;
```

Comment:
This one has `""` in the code, so possibly we could extract a type for it to share between `geneLevel` and `scatterplotLevel`?

E.g.,

```ts
type Level = "bottom" | "top" | ""

scatterplotLevel: Level;
geneLevel: Level;
```

```suggestion
  geneLevel: "top" | "bottom" | "";
```